args_builder.go

```diff
@@ -0,0 +1,64 @@
+package gonymizer
+
+import (
+	"fmt"
+	"strings"
+)
+
+
+func CreateDumpArgs(
+	conf PGConfig,
+	dumpfilePath,
+	schemaPrefix string,
+  excludeTables,
```

Comment:
nit: looks like tabbing may be off for some of the lines in this file

Reply:
ah, so my files locally have spaces, so something funky is going on here, i did wonder about that in the comparison!, i'll investigate

Reply:
@junkert think i've resolved the tab issues? its a bit odd because its very different on github vs local, please let me know if the tabs are still wrong anywhere and i'll work it out! i've at the least removed the mixture of tabs and spaces i committed before !!!